src/components/shop/item-registry.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * @copyright BBC 2020
+ * @author BBC Children's D+E
+ * @license Apache-2.0
+ */
+
+import fp from "../../../lib/lodash/fp/fp.js";
+
+export const initRegistry = itemsArray => {
+    const items = fp.clone(itemsArray);
+
+    const getResult = result => {
```

Comment:
Can use implicit return to make this one line if you prefer.

Reply:
5e992d4